### PR TITLE
*: Fix issue 1118

### PR DESCRIPTION
**Pull request by zimulala**
fix issue #1118

### CODE REVIEW

File: executor/executor_write.go [hunk 1]
```diff
@@ -714,12 +749,9 @@ func (e *InsertValues) initDefaultValues(row []types.Datum, marked map[int]struc
 				return errors.Trace(err)
 			}
 			row[i].SetInt64(recordID)
-			// Notes: incompatible with mysql
-			// MySQL will set last insert id to the first row, as follows:
-			// `t(id int AUTO_INCREMENT, c1 int, PRIMARY KEY (id))`
-			// `insert t (c1) values(1),(2),(3);`
-			// Last insert id will be 1, not 3.
-			variable.GetSessionVars(e.ctx).SetLastInsertID(uint64(recordID))
+			if e.currRow == 0 {
```
Comment (shenli) [inline executor/executor_write.go]: Please add a few comments here.

File: executor/executor_write.go [hunk 2]
```diff
@@ -676,6 +680,33 @@ func (e *InsertValues) fillRowData(cols []*table.Column, vals []types.Datum) ([]
 	return row, nil
 }
 
+func (e *InsertValues) setLastInsertID() {
```
Comment (shenli) [inline executor/executor_write.go]: Please add a few comments for this function.

**Comment by shenli** [comment]
Reset LGTM

**Comment by zimulala** [comment]
PTAL @shenli @coocood

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**